app/client/components/pages/dashboardPage.jsx

```diff
@@ -149,7 +149,7 @@ class DashboardPage extends React.Component {
                 defaultTimeSeries="all_payments_volume"
                 filterObject="credit_transfer"
                 timeSeriesNameLabels={[
-                  ["all_payments_volume", "Volume"],
+                  ["all_payments_volume", "Payment Volume"],
```

Comment:
Does #429 change this? Does Volume mean all transactions on the system or just internal payments? If only internal payments then I think payment volume is better.

Reply:
@tristanhcole It is all payments! What verbiage would you suggest to use here?

Reply:
the tooltips _may_ help with this. Part of the reason this one is hard is that `Volume` on its own is concise and correct.